auth_backends/backends.py

```diff
@@ -99,6 +99,24 @@ def _map_user_details(self, response):
         return dest
 
 
+class EdXSettingsAwareOpenIdConnect(EdXOpenIdConnect):
```

Comment:
I believe these changes can be applied directly to `EdXOpenIdConnect`. There is no longer a need for the child class since we have backwards-compatibility.